README.md

```diff
@@ -1,137 +1,123 @@
 
-Any iteration on the stated purpose, initiatives, and supporting members should continue as needed until endorsement is received from at least one member of the Editorial Team. Submissions that fail to receive endorsement may be removed by Solid Administrators after six months.
+The panel proposal can be revised until at least one member of the Editorial Team endorses the proposal. Submissions that fail to receive endorsement may be removed by Solid Administrators after 3 months.
 
-Any Editor endorsing a panel is expected to attend regular sessions, and to provide the Panel Chairs and Panel with insight on specification priorities, as well as support work on any relevant initiatives in flight.
+Any Editor endorsing a panel is expected to attend the panel regularly, and to provide the panel insight on specification priorities and support the work of the panel.
 
-Each panel is chaired by one or more Panel Chairs. An Editorial Team member endorsing a panel is responsible for electing the Panel Chairs, barring any vetoes from another Editor, another Panel Chair of the same panel, or vetoes from a majority of panel members. The Solid Director reserves the right to elect and/or change Panel Chairs. In the event that an Editor should be nominated as a Panel Chair, they must be elected by another member of the Editorial Team.
+Each panel is chaired by one or more Panel Chairs. An Editorial Team member endorsing a panel is responsible for appointing the Panel Chairs, barring any objections from another Editor, another Panel Chair of the same panel, or objection from a majority of panel members. The Solid Director reserves the right to appoint and/or change Panel Chairs. In the event that an Editor is nominated as a Panel Chair, they must be appointed by another member of the Editorial Team.
 
  Responsibilities of a Panel Chair include but are not limited to:
 
 - Coordinating and communicating panel initiatives
-- Collaboaring with Editors on priority and direction of panel initiatives
+- Working with Editors on prioritizing panel initiatives
 - Ensuring panel meetings are focused on advancing panel initiatives
-- Ensuring every Panel meeting has a clear goal and agenda
+- Ensuring every panel meeting has a clear goal and agenda
 - Ensuring all actions arising from meetings are tracked to completion
-- Ensuring target dates are set for outcomes
-- Working with Panel members to achieve broad consensus on decisions, with a consistent aim towards unanimity, in general accordance with the W3C process for realizing [consensus](https://www.w3.org/2019/Process-20190301/#Consensus).
-- Referring to the W3C process to [constructively manage dissent](https://www.w3.org/2019/Process-20190301/#managing-dissent) from panel members so that the group can continue to make progress in the absence of broad consensus.
-- Ensuring panel decisions, activities, and achievements are tracked and communicated.
+- Ensuring target dates are set for objectives
```

Comment:
If actions from meetings are rarely carried out and if target dates for objectives are never met, what happens then?
Should a panel be dissolved if it does not make progress after a given period of time? I know if there is no activity then it can be removed but no progress is almost worse as it means there is work to be done and the group involved is not getting it done?

What about panels that started with 5 or 6 people but quickly get to a point where only a couple of people attend  or do any work? Should that panel continue? Does it indicate there is not sufficient interest in the topic of discussion?

Reply:
Perhaps, but I'm less worried about garbage collecting panels if they are non-formal groups anyway. We create them here, but they are given basically no powers / authority. So if they stagnate, there's no harm, though it would be good to remove them. I've added a line that says that the editors can close a panel due to inactivity by a majority vote of the editors.